pkg/settings/integration_tests/integration_tests_test: TestSettingsPersistenceEndToEnd failed

pkg/settings/integration_tests/integration_tests_test.TestSettingsPersistenceEndToEnd [failed](https://mesolite.cluster.engflow.com/invocations/default/e1c7e925-4616-4913-bc0b-b76c1f9f9ea5?testReportRun=1&testReportShard=2&testReportAttempt=1#targets-Ly9wa2cvc2V0dGluZ3MvaW50ZWdyYXRpb25fdGVzdHM6aW50ZWdyYXRpb25fdGVzdHNfdGVzdA==) on release-24.3 @ [92e32a1be43ac3cd3a9a573a6d2928d7e3429aaf](https://github.com/cockroachdb/cockroach/commits/92e32a1be43ac3cd3a9a573a6d2928d7e3429aaf):


```
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:416: condition failed to evaluate within 45s: from settings_test.go:419: not updated yet: expecting something, got 
    --- FAIL: TestSettingsPersistenceEndToEnd/different_node (45.17s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=2</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/unowned
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSettingsPersistenceEndToEnd.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-43737

pkg/settings/integration_tests/integration_tests_test.TestSettingsPersistenceEndToEnd [failed](https://tanzanite.cluster.engflow.com/invocations/default/d1a2e0a5-aeba-4fe1-84e8-dbb17a439c0d?testReportRun=25&testReportShard=2&testReportAttempt=1#targets-Ly9wa2cvc2V0dGluZ3MvaW50ZWdyYXRpb25fdGVzdHM6aW50ZWdyYXRpb25fdGVzdHNfdGVzdA==) on release-24.3 @ [b1c0838dab38a0b50c2c9197c1c2d5588501758c](https://github.com/cockroachdb/cockroach/commits/b1c0838dab38a0b50c2c9197c1c2d5588501758c):


```
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:420: not updated yet: expecting something, got 
    settings_test.go:416: condition failed to evaluate within 45s: from settings_test.go:419: not updated yet: expecting something, got 
    --- FAIL: TestSettingsPersistenceEndToEnd/different_node (45.16s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=25</code>
 - <code>shard=2</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #136957 pkg/settings/integration_tests/integration_tests_test: TestSettingsPersistenceEndToEnd failed [C-test-failure O-robot P-2 T-db-server branch-release-24.2.6-rc]
- #136474 pkg/settings/integration_tests/integration_tests_test: TestSettingsPersistenceEndToEnd failed [C-test-failure O-robot P-2 T-db-server branch-release-23.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSettingsPersistenceEndToEnd.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/settings/integration_tests/BUILD.bazel
+++ b/pkg/settings/integration_tests/BUILD.bazel
@@ -7,6 +7,7 @@ go_test(
         "propagation_test.go",
         "settings_test.go",
     ],
+    exec_properties = {"test.Pool": "large"},
     shard_count = 6,
     deps = [
         "//pkg/base",

